### ISSUE TITLE
Can't specify intermediate time points in a request

### ISSUE THREAD

**Issue by mpu-creare** [creare-com/podpac#226]
**Description**
When I request a time range as a specified frequency, the SMAP node retrieves all intermediate times as well (Very inefficient). 

**Steps to Reproduce**
```
import podpac
smap = podpac.datalib.SMAP(interpolation='nearest')
bounds = podpac.Coordinates([
    podpac.clinspace(49.25, 39.75, 2, 'lat'), 
    podpac.clinspace(-116.75, -95.25, 2, 'lon')])
coords = smap.shared_coordinates.intersect(bounds)
coords2017 = podpac.coordinates.merge_dims([podpac.Coordinates([podpac.crange('2017-05-01T12', '2017-09-12T12', '12,h', 'time')]),
                                            coords])
smap.eval(coords2017)
```

I have logging turned on, and this produces the following: 

```bash
INFO:pydap:Fetching URL: https://n5eil02u.ecs.nsidc.org/opendap/SMAP/SPL4SMAU.004/2017.05.01/SMAP_L4_SM_aup_20170501T120000_Vv4030_001.h5.dods?Analysis_Data_sm_surface_analysis[196:1:291][677:1:907]
INFO:pydap:Fetching URL: https://n5eil02u.ecs.nsidc.org/opendap/SMAP/SPL4SMAU.004/2017.05.01/SMAP_L4_SM_aup_20170501T150000_Vv4030_001.h5.dods?Analysis_Data_sm_surface_analysis[196:1:291][677:1:907]
INFO:pydap:Fetching URL: https://n5eil02u.ecs.nsidc.org/opendap/SMAP/SPL4SMAU.004/2017.05.01/SMAP_L4_SM_aup_20170501T180000_Vv4030_001.h5.dods?Analysis_Data_sm_surface_analysis[196:1:291][677:1:907]
INFO:pydap:Fetching URL: https://n5eil02u.ecs.nsidc.org/opendap/SMAP/SPL4SMAU.004/2017.05.01/SMAP_L4_SM_aup_20170501T210000_Vv4030_001.h5.dods?Analysis_Data_sm_surface_analysis[196:1:291][677:1:907]
...
```
Note that I'm also getting 3, 6, and 9 o'clock

**Expected Behavior**
Log should look like this: 
```bash
INFO:pydap:Fetching URL: https://n5eil02u.ecs.nsidc.org/opendap/SMAP/SPL4SMAU.004/2017.05.01/SMAP_L4_SM_aup_20170501T120000_Vv4030_001.h5.dods?Analysis_Data_sm_surface_analysis[196:1:291][677:1:907]
INFO:pydap:Fetching URL: https://n5eil02u.ecs.nsidc.org/opendap/SMAP/SPL4SMAU.004/2017.05.02/SMAP_L4_SM_aup_20170502T120000_Vv4030_001.h5.dods?Analysis_Data_sm_surface_analysis[196:1:291][677:1:907]
INFO:pydap:Fetching URL: https://n5eil02u.ecs.nsidc.org/opendap/SMAP/SPL4SMAU.004/2017.05.03/SMAP_L4_SM_aup_20170503T120000_Vv4030_001.h5.dods?Analysis_Data_sm_surface_analysis[196:1:291][677:1:907]
INFO:pydap:Fetching URL: https://n5eil02u.ecs.nsidc.org/opendap/SMAP/SPL4SMAU.004/2017.05.04/SMAP_L4_SM_aup_20170504T120000_Vv4030_001.h5.dods?Analysis_Data_sm_surface_analysis[196:1:291][677:1:907]
...
```

**Comment by mpu-creare** [comment]
This will be fixed in #430

**Comment by mpu-creare** [comment]
Actually need to pass selector to the compositor selectors... TODO

**Comment by mpu-creare** [comment]
This has been fixed in #434
